### ISSUE TITLE
Linux node crashing: "route ip+net: netlinkrib: too many open files"

### ISSUE THREAD

**Issue by ZuZuD** [idena-network/idena-go#608]
Despite having increased the maximum number of open file descriptors

```
root@vultr[~]> ulimit -Hn
65535
root@vultr[~]> ulimit -Sn
65535
```

my ubuntu 20.10 node crashes regularly with the following error:

```
2020-12-20T07:54:22.084Z	ERROR	basichost	failed to resolve local interface addresses	{"error": "route ip+net: netlinkrib: too many open files"}
2020-12-20T18:56:05.645Z	ERROR	basichost	failed to resolve local interface addresses	{"error": "route ip+net: netlinkrib: too many open files"}
2020-12-21T19:16:36.746Z	ERROR	basichost	failed to resolve local interface addresses	{"error": "route ip+net: netlinkrib: too many open files"}
2020-12-23T17:21:41.703Z	ERROR	basichost	failed to resolve local interface addresses	{"error": "route ip+net: netlinkrib: too many open files"}
2020-12-27T12:42:43.326Z	ERROR	basichost	failed to resolve local interface addresses	{"error": "route ip+net: netlinkrib: too many open files"}
```

This is the node version I run:
```
root@vultr[~]> ./idena-node-linux-latest -v
idena-node-linux-latest version 0.24.0
```
Once it happened, the DB is corrupted. Any recommendation on how to fix this?

**Comment by ridenaio** [comment]
Hi, we've resolved a similar DB corruption problem in `0.24.2`. Please, update your node and write again if DB corruption still exists.

**Comment by ltraveler** [comment]
I just got the same problem with the latest idena node 0.24.3
